src/Numerics/Mesh/Filters.jl

```diff
@@ -160,6 +161,66 @@ struct ExponentialFilter <: AbstractSpectralFilter
     end
 end
 
+"""
+    BoydVandevenFilter(grid, Nc=0, s=32, α=-log(eps(eltype(grid))))
+
+Returns the spectral filter using the logorithmic error function of
+the form:
+```math
+σ(η) = 1/2 erfc(2*sqrt(s)*χ(η)*(abs(η)-0.5))
+```
+whenever s ≤ i ≤ N, and 1 otherwise. The function `χ(η)` is defined
+as
+```math
+χ(x) = sqrt(-log(1-4*(abs(x)-0.5)^2)/4*(abs(x)-0.5)^2)
+```
+if `x != 0.5` and `1` otherwise. Here, `s` is the filter order,
+the filter starts with polynomial order `Nc`, and `alpha` is a parameter
+controlling the smallest value of the filter function.
+
+### References
+
+    @inproceedings{boyd1996erfc,
+    title={The erfc-log filter and the asymptotics of the Euler and Vandeven sequence accelerations},
+    author={Boyd, JP},
+    booktitle={Proceedings of the Third International Conference on Spectral and High Order Methods},
+    pages={267--276},
+    year={1996},
+    organization={Houston Math. J}
+    }
+"""
+struct BoydVandevenFilter <: AbstractSpectralFilter
+    "filter matrix"
+    filter
+
+    function BoydVandevenFilter(
+        grid,
+        Nc = 0,
+        s = 32,
+        α = -log(eps(eltype(grid))),
+    )
+        AT = arraytype(grid)
+        N = polynomialorder(grid)
+        ξ = referencepoints(grid)
+
+        @assert iseven(s)
+        @assert 0 <= Nc <= N
+
+        σ(η) = 0.5*erfc(2*sqrt(s)*χ(η)*(abs(η)-0.5))
```

Comment:
The expressions here and in `χ` simplify if you substitute `\bar η = abs(η)-0.5` and make `χ` a function of this transformed `η`.